app/src/main/java/com/timothyolt/evolutionarydesign/album/AlbumActivity.kt

```diff
@@ -0,0 +1,73 @@
+package com.timothyolt.evolutionarydesign.album
+
+import android.os.Bundle
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.lifecycleScope
+import androidx.recyclerview.widget.RecyclerView
+import com.timothyolt.evolutionarydesign.image.Image
+import com.timothyolt.evolutionarydesign.image.ImageAdapter
+import com.timothyolt.evolutionarydesign.R
+import com.timothyolt.evolutionarydesign.requireInjector
+import kotlinx.coroutines.*
+import org.json.JSONObject
+import java.io.BufferedInputStream
+import java.net.HttpURLConnection
+import java.net.URL
+
+class AlbumActivity : AppCompatActivity() {
+
+    data class Dependencies(val albumId: String)
+
+    private lateinit var dependencies: Dependencies
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        dependencies = requireInjector().inject(this)
+        setContentView(R.layout.activity_main)
+        val adapter = ImageAdapter()
+        findViewById<RecyclerView>(R.id.images).adapter = adapter
+
+        lifecycleScope.launch {
```

Comment:
Not sure what our refresh plans are at this point, but maybe we can just extract this block into a function so that it's easy to move the retrieval site.